Basket-Donation-Payments/server.js

```diff
@@ -10,25 +10,28 @@ const stripe = require("stripe")(process.env.STRIPE_SECRET_KEY);
 app.use(express.static("."));
 app.use(express.json());
 
-const calculateOrderAmount = items => {
+const calculateTransferAmount = totalAmount, basketItems => {
```

Comment:
This should be a separate pull request, maybe a separate repository as you will need to deploy this later, right?

Reply:
Oops, I haven't quite thought about deployment things yet. For now, I'm running the Express/Node server locally so my payments go through.
Eventually, though, I'll have to set up Heroku for deployment & I'll make that a separate PR - will configure when I get there.

Do you think that sounds doable? Or are there any steps I'm missing in the interim?

Reply:
That sounds good, but you can still setup a separate repo now to keep git commit history clean